docs/concepts/duplicate-legacy-metaos-add-ins.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Managing both a unified manifest and an XML manifest version of your Office Add-in
+description: Learn when and how to maintain versions of your add-in for each type of manifest.
+ms.topic: best-practice
+ms.date: 07/11/2023
+ms.localizationpriority: medium
+---
+
+# Managing both a unified manifest and an XML manifest version of your Office Add-in
+
+After you've created a version of your add-in that uses the [unified manifest for Microsoft 365](../develop/json-manifest-overview.md), you need to decide whether to maintain the existing add-in and, if so, what relationship it will have with the new version. In the long run, the best practice is to replace the existing add-in with the new version, but initially add-ins that support the unified manifest can only be installed on Microsoft 365 version 2309 ?????.????? and later. We're working hard to bring support to older versions of Microsoft 365. In the meantime, you need to maintain both versions.
+
+> [!TIP]
+> For information about converting an existing add-in to use the unified manifest, see [Convert an add-in to use the unified manifest for Microsoft 365](../develop/convert-xml-to-json-manifest.md).
+
+There are also some scenarios where you need to maintain both for a longer period, including the following.
+
+- Even after support for the unified manifest is available in older versions of Microsoft 365, there will be a further period in which add-ins that use it won't be installable on perpetual versions of Office, such as volume-licensed perpetual versions of Office 2021 (and earlier).
```

Comment:
Do we need to say this, or can we push it under the banner of "versions that do not support it"? (i.e., are we overemphasizing this point?)